x-pack/plugin/eql/qa/common/src/main/java/org/elasticsearch/test/eql/CommonEqlRestTestCase.java

```diff
@@ -31,6 +31,11 @@
     private static final String defaultValidationIndexName = "eql_search_validation_test";
     private static final String validQuery = "process where user = 'SYSTEM'";
 
+    @After
+    public void checkSearchContent() throws Exception {
```

Comment:
Add assertion that all search contexts are closed after each single test.

Reply:
Really minor: `checkSearchContent` or `checkSearchContext`?